fileteasend/app.py

```diff
@@ -0,0 +1,169 @@
+        send_url = urljoin(self.url, token)
+        with open(file_to_send, 'rb') as f:
+            send_response = self.session.put(send_url, f)
+        logger.debug(send_response)
+
+    def receive_response(self, peer_id, file_to_send):
+        """
+        Wait for server response to send file
+        """
+        logger = logging.getLogger('filetea.FileTeaClient.receive_response')
+        receive_url = urljoin(self.url, 'transport/lp/receive')
+        receive_response = self.session.get(receive_url, params=peer_id)
+        receive_response_text = receive_response.text
+        logger.debug(receive_response_text)
+        try:
+            receive_response_text = receive_response_text[1:]
+            receive_response_json = json.loads(receive_response_text)
+            logger.debug('Decoded JSON response: {}'.format(receive_response_json))
+            if receive_response_json['method'] == 'fileTransferNew':
+                token = receive_response_json['params'][-1]
+                logger.debug('Got a file transfer request from token {}'.format(token))
+                print('File transfer request received from token {}'.format(token))
+                self.send_file(file_to_send, token)
+            else:
+                logger.info('Not a file transfer request')
+        except Exception as e:
+            logger.info('Not a file transfer request: {}'.format(e))
+            pass
+
+def exit(signal, frame):
+    sys.exit(0)
+
+def run(args):
+    logger = logging.getLogger('filetea')
+    # Set FileTea server URL
+    server_url = args.url if args.url != None else os.getenv('FILETEAURL', 'https://filetea.me/')
+    if not urlparse(server_url).netloc:
+        sys.exit('No schema supplied on URL.')
+
+    file_tea_client = FileTeaClient(server_url)
+    # Get a peer-id registered with endpoint
+    peer_id = file_tea_client.get_peer_id()
+    # Register specified file with endpoint, which returns the URL to download the file
+    file_url = file_tea_client.register_file(peer_id, args.file)
+    logger.info('URL for file {}: {}'.format(args.file, file_url))
+    print('URL: {}'.format(file_url))
+    print('Press CTRL+C to stop sharing...')
+
+    while True:
+        # Main loop
+        logger.info('Waiting for a server response')
+        file_tea_client.receive_response(peer_id, args.file)
+
+def main():
+    signal.signal(signal.SIGINT, exit)
+    parser = argparse.ArgumentParser(description='Easy file transfer using the FileTea service')
+    parser.add_argument('--url', '-l', help='filetea server url (also FILETEAURL)')
+    parser.add_argument('--verbose', '-v', action='count', default=0, help='be verbose (vv to increase verbosity)')
+    parser.add_argument('file', help='file to send')
+    parser.set_defaults(func=run)
+    args = parser.parse_args()
+
+    # Set loglevel equivalents
+    log_levels = {
+            0: logging.ERROR,
+            1: logging.INFO,
+            2: logging.DEBUG
+                 }
```

Comment:
Same style nitpick as above, I'd prefer this closing bracket to align with the main indentation level, not the opening bracket.